view/frontend/templates/express-pay.phtml

```diff
@@ -0,0 +1,37 @@
+<?php
+use Magento\Framework\View\Element\Template;
```

Comment:
Strict typing should be enabled for this template and there should be an empty line after the opening PHP tag.

Reply:
Added